src/views/inputHandler.js

```diff
@@ -0,0 +1,26 @@
+import { Console } from '@woowacourse/mission-utils';
+import { GAME_MESSAGES, ERROR_MESSAGES } from '../constants/index.js';
+import { validateTryCount } from '../utils/index.js';
+
+export const inputHandler = {
+  carNameInput: async () => {
+    const carNames = await Console.readLineAsync(GAME_MESSAGES.START);
+
+    if (carNames.trim() === '') {
+      throw new Error(ERROR_MESSAGES.CAR_ERROR_MESSAGES.BLANK);
+    }
+    return carNames;
+  },
```

Comment:
저도 우혁님 의견처럼 분리하는 게 더 좋을 것 같다는 생각이 들어요!

```suggestion
export const inputHandler = {
  carNameInput: async () => {
    const carNames = await Console.readLineAsync(GAME_MESSAGES.START);
    return validateInput(input);
  },
```

Reply:
오류 처리 부분을 빼야하는게 더 깔끔해 보이긴 하네요 감사합니다 !